cpp/src/prims/edge_partition_src_dst_property.cuh

```diff
@@ -94,6 +94,8 @@ class edge_partition_major_property_device_view_t {
     }
 
     assert((partition_idx == size_t{0}) || edge_partition_major_value_start_offsets_);
+    // This is failing in the debug build...
+    //    assert((partition_idx == size_t{0}) || edge_partition_value_firsts_);
```

Comment:
This code is deleted in one of the previously merged PRs. Please merge with the current branch.